### DISCUSSION TITLE
Any interest in publishing UX mockup components (ie figma or XD outputs)?

### DISCUSSION THREAD

**Discussion by charlesabarnes** [bullhorn/novo-elements#1428] [General]
Ocassionally, I work on and contribute to a custom application that uses Bullhorn's novo-elements.  I would like to sketch out ideas for new features and functionality.   If that isn't something Bullhorn is currently interested in, that is fine. I can spin that stuff up on my end,

Thanks!

**Comment by blakefosdick** [comment]
Did you ever create or find any Novo components for Figma or XD ?
